modules/plugin/epsg-hsql/src/test/resources/org/geotools/referencing/esri_tests.properties

```diff
@@ -0,0 +1,36 @@
+2008=PROJCS["NAD_1927_CGQ77_MTM_2_SCoPQ",GEOGCS["GCS_NAD_1927_CGQ77",DATUM["D_NAD_1927_CGQ77",SPHEROID["Clarke_1866",6378206.4,294.9786982]],PRIMEM["Greenwich",0.0],UNIT["Degree",0.0174532925199433]],PROJECTION["Transverse_Mercator"],PARAMETER["False_Easting",304800.0],PARAMETER["False_Northing",0.0],PARAMETER["Central_Meridian",-55.5],PARAMETER["Scale_Factor",0.9999],PARAMETER["Latitude_Of_Origin",0.0],UNIT["Meter",1.0]]
```

Comment:
Gotcha, I am not especially worried since this is test-data and not distributed with the library for use in downstream applications.

I imagine we will gather up additional test-data from ESRI/Oracle/Others; more test data can only improve compatibility with these projects.